### DISCUSSION TITLE
Good ODSC Webinar "Generate Synthetic Tabular Data with GANs", are you Open to collaboration?

### DISCUSSION THREAD

**Discussion by Marc-Cox-08** [lady-h-world/My_Garden#11] [Open End Discussions]
Thank you for Good ODSC Webinar "Generate Synthetic Tabular Data with GANs", are you Open to collaboration? 
Asking here with new comment as it appears "Leave messages here for collaboration opportunities 💝 #5" was closed.

**Comment by lady-h-world** [comment]
Thank you @Marc-Cox-08 ! Currently I'm still very busy, but may I ask what kind of collaboration?
